build.mjs

```diff
@@ -0,0 +1,61 @@
+import { build } from 'esbuild';
+import * as process from 'node:process'
+
+
+const buildOptions = {
+    // Produce the build results in the dist folder
+    outdir: 'dist',
+
+    // Map source code typescript files to their output files (dist/*.js)
+    entryPoints: [
+        { in: 'src/index.ts', out: 'index' },
+        { in: 'src/s3.ts', out: 's3' },
+        { in: 'src/secrets-manager.ts', out: 'secrets-manager' },
+        { in: 'src/sqs.ts', out: 'sqs' },
+        { in: 'src/ssm.ts', out: 'ssm' },
+        { in: 'src/kms.ts', out: 'kms' },
+        { in: 'src/kinesis.ts', out: 'kinesis' },
+        { in: 'src/event-bridge.ts', out: 'event-bridge' },
+        { in: 'src/lambda.ts', out: 'lambda' },
+        { in: 'src/signature.ts', out: 'signature' },
+    ],
+
+    // k6 supports the ES module format, and using it avoids transpiling and leads
+    // to faster time to start a test, and better overall test performance.
+    format: 'esm',
+
+    // k6 JS runtime is browser-like.
+    platform: 'browser',
+
+    // Bundle all dependencies into the output file(s)
+    bundle: true,
+
+    // Generate source maps for the output files
+    sourcemap: true,
+
+    // Allow importing modules from the 'k6' package, all its submodules, and
+    // all HTTP(S) URLs (jslibs).
+    external: [
+        'k6', // Mark the 'k6' package as external
+        'k6/*',       // Mark all submodules of 'k6' as external
+        "/^https?://" // Regex to mark all HTTP imports as external
```

Comment:
Q: Should this be tied to `jslib.k6.io`? Or do we need imports through HTTP other than these? 🤔